pkg/services/sqlstore/user.go

```diff
@@ -186,25 +184,6 @@ func (ss *SQLStore) createUser(ctx context.Context, sess *DBSession, args userCr
 	return user, nil
 }
 
-func (ss *SQLStore) CloneUserToServiceAccount(ctx context.Context, siUser *models.SignedInUser) (*models.User, error) {
```

Comment:
question: I'm guessing this functionality ended up not being used?

Reply:
It was reimplemented in the SA store and we have cordoned it off for now